static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-support-baseline.js

```diff
@@ -60,6 +67,26 @@ const isEpicWithinViewLimit = (test: EpicABTest): boolean => {
     return withinViewLimit && enoughDaysBetweenViews;
 };
 
+const changeLinks = (cssClass: string, link: string) => {
+    [...document.getElementsByClassName(cssClass)].forEach(el => {
+        el.href = link;
+    });
+};
+
+const changeHeaderLinks = (
+    becomeSupporterLink: string,
+    subscribeLink: string
+) => {
+
+    changeLinks('js-become-member', becomeSupporterLink);
+    changeLinks('js-subscribe', subscribeLink);
+
+    changeLinks('js-become-member-new-header', becomeSupporterLink);
```

Comment:
In that case, I think a class should be added like `js-change-member-link` and `js-change-subscribe-link`. 

The principle behind that is the class should describe what its doing so there shouldn't be any surprises.